### PR TITLE
Allow only one child in Sortable

### PR DESCRIPTION
**Pull request by vizath**
this.props.children may be an array or an object (see: https://facebook.github.io/react/tips/children-props-type.html)

### CODE REVIEW

**Comment by vizath** [comment]
Sorry, it should have been 2 pull requests, but I made the pull request from my master :(

**Comment by jasonslyvia** [comment]
Thank you for you contribution @vizath , I'll take a look of your PR ASAP.

File: lib/index.js [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
+ * @file react-anything-sortable
+ * @author jasonslyvia
+ */
 
-var _interopRequireWildcard = function (obj) { return obj && obj.__esModule ? obj : { 'default': obj }; };
+/*eslint new-cap:0, consistent-return: 0 */
+
+'use strict';
 
 Object.defineProperty(exports, '__esModule', {
```
Comment (jasonslyvia) [inline lib/index.js]: In order to make `react-anything-sortable` work in IE 8, you can't use `Object.defineProperty` for non-DOM elements. Maybe you should take a look of `Babel`'s `loose mode` configuration for better compiling options.

**Comment by vizath** [comment]
I added the loose option in babel, but there seems to still have a lot of differences in the build. Maybe I use a newer version of babel ? Or I miss a couple of options ? 
You should specifiy the required options for compiling in the gulp file. This way, anybody building the project will get the same parameters.

**Comment by jasonslyvia** [comment]
My bad, let me handle this compiling issue.

2015-05-29 11:27 GMT+08:00 Maxime Fafard <notifications@github.com>:

**Comment by jasonslyvia** [comment]
Seems good, merged into master. Thanks again @vizath .

**Pull request merged**